### ISSUE TITLE
Viewing a record type in `ucm` does not provide information about argument names

### ISSUE THREAD

**Issue by samgqroberts** [unisonweb/unison#1698]
Let's say I have a record type that I've `add`ed to `ucm` based on this definition:
```
type Rectangle = { width: Nat, height: Nat }
```
Thereafter if I try using `find` and `view` I cannot retrieve information about the argument names that the record type was created with (here, `width` and `height`). 
```
.> find Rectangle
1. type Rectangle
2. Rectangle.Rectangle : Nat -> Nat -> Rectangle
3. Rectangle.height : Rectangle -> Nat
4. Rectangle.height.modify : (Nat ->{𝕖 } Nat) -> Rectangle ->{} Rectangle
5. Rectangle.height.set : Nat -> Rectangle -> Rectangle
6. Rectangle.width : Rectangle -> Nat
7. Rectangle.width.modify : (Nat ->{𝕖 } Nat) -> Rectangle ->{} Rectangle
8. Rectangle.width.set : Nat -> Rectangle -> Rectangle
.> view 1
type Rectangle = Rectangle Nat Nat
.> view 2
type Rectangle = Rectangle Nat Nat
```

I believe the desired solution here may be to update the outputs of `view 1` (and perhaps `view 2`) in this example to provide information about the argument names, specifically to annotate the first type constructor argument with `width` and the second with `height`.

**Comment by aryairani** [comment]
I guess ucm could regenerate getters and setters for a type, find their hashes, see if the hashes exist in the namespace, see what names they already have; and then if all the pieces are in place, pretty-print the type with record declaration syntax.

**Comment by pchiusano** [comment]
There is already some logic for this in the pretty printer and I have seen it work before. So I am not sure why it's not working here.

**Comment by pchiusano** [comment]
This has been fixed recently in trunk! By #2403

**Comment by samgqroberts** [comment]
Go Unison team! 🎉
